SnakesNLadders/src/services/Player.java

```diff
@@ -0,0 +1,54 @@
+package services;
+
+
+import models.BoardEntity;
+import models.PlayerEntity;
+
+public class Player {
+	PlayerEntity player;
+	public Player(PlayerEntity player)
+	{
+		this.player=player;
+	}
+	public void play(BoardEntity board)
+	{
+		int diceThrow=board.getDice().run(),diceThrow2nd=0,diceThrow3rd=0,oldPosition=board.getPlayerPositions().get(this),newPosition;
```

Comment:
All of the logic in the play function should ideally not be with the player but a service that belongs to the game or board.